packages/mobile/components/buttons/ProfileButton.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="typescript">
```

Comment:
I would try improving a bit the CSS of this button, I would set it so the profile name is always in one line and it truncates with ellipsis ... if it reaches the end of the available space
![image](https://user-images.githubusercontent.com/3624944/194311818-3340ae55-a10a-48b7-bc4c-1885efa1cfd9.png)